bot/cogs/todo_list.py

```diff
@@ -0,0 +1,161 @@
+from asyncio import sleep
+from bot.settings import BASE_DIR
+from discord.ext import tasks
+from discord.ext.commands import Bot, Cog, command
+import json
+
+
+class TodoList(Cog):
```

Comment:
Overall the command is quite slow, this is not terrible, but may need optimization. Works really well overall